docs/guides/plugins/presale-saml.md

```diff
@@ -1,215 +1,243 @@
+```
 
-!!! Note 
-    If you are a customer of our hosted [pretix.eu](https://pretix.eu) offering, the menu will only appear once one of our friendly customer service agents has enabled the Presale SAML Authentication plugin for at least one of your events. 
-    Feel free to get in touch with us!
+If you want every organizer to provide their own SP, extend the configuration file `<config>` with the following lines instead: 
 
-### Setting up the SP
+``` ini
+[presale-saml]
+level=organizer
+```
 
-No matter where your SP configuration lives, you will be greeted by a very long list of fields of which almost all of them will need to be filled. 
-Please don't be discouraged - most of the settings don't need to be decided by yourself and/or are already preset with a sensible default setting.
+If you set `level` to `global`, then your instance will use a single system-wide SP. 
+If you set `level`to `organizer`, then the SP will have to be configured for each organizer individually. 
+If you do not define `level`, then it will default to `organizer`. 
 
-If you are not sure what setting you should choose for any of the fields, you should reach out to your IdP operator as they can tell you exactly what the IdP expects and - more importantly - supports.
+### Plugin activation 
 
-`IdP Metadata URL`
+<!-- md:hosted --> 
 
-"Please provide the URL where your IdP outputs its metadata. 
-For most IdPs, this URL is static and the same for all SPs. 
-If you are a member of the DFN-AAI, you can find the metadata for the [Test-, Basic- and Advanced-Federation](https://doku.tid.dfn.de/en:metadata) on their website. 
-Please do talk with your local IdP operator though, as you might not even need to go through the DFN-AAI and might just use your institution's local IdP which will also host their metadata on a different URL.
+If you are using pretix Hosted, the pretix team has to make the "Presale SAML Authentication" plugin available for your organizer account before you can use it. 
+Contact support via [email](mailto:support@pretix.eu) or [phone](tel:+4962213217750) and ask them to activate the SAML plugin. 
+Once the plugin has been made available for the organizer account, the button ":fa3-key: SAML" will appear under the organizer settings in the sidebar menu. 
 
-    The URL needs to be publicly accessible, as saving the settings form will fail if the IdP metadata cannot be retrieved. 
-    pretix will also automatically refresh the IdP metadata on a regular basis.
+<!-- md:enterprise --> 
 
-`SP Entity Id`
+If you are using pretix Enterprise and have access to an admin account, you have to make the "Presale SAML Authentication" plugin available for your organizer account. 
+Click the :btn-icon:fa3-id-card: Admin mode: button. 
+Navigate to :navpath:Your organizer: → :fa3-wrench: Settings: and open the :btn:General: tab. 
+Under "Allow usage of restricted plugins", check the box next to "Presale SAML Authentication" and click the :btn:Save: button. 
```

Comment:
The plugin is only restricted on pretix hosted - for self-hosters, it's just like any other plugin and can be activated directly.